apis/quay/v1/quayregistry_types.go

```diff
@@ -68,6 +68,12 @@ var requiredComponents = []ComponentKind{
 	ComponentRedis,
 }
 
+var supportsVolumeOverride = []ComponentKind{
+	ComponentPostgres,
+	ComponentClair,
+	ComponentObjectStorage,
```

Comment:
Does ObjectStorage support Override ? The Jira card has, as an acceptance criteria:

"volume overrides available for every managed component that uses a PersistentVolume (**Noobaa Backend**, Postgres for Quay, Postgres for Clair)" 

I  know that we do not set the size of the Nooba Backend but before merging this we better clarify the behavior with the Jira card reporter.